examples/dgmulti_2d/elixir_mhd_weak_blast_wave.jl

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
Good question. Yes, I think the CFL factors are different, we tend to use a fairly conservative estimate proportional to `O(h / N^2)`, whereas I think the `TreeMesh` estimate is `O(h / N)` with a larger constant. This should also mean that we can bump up the CFL for `DGMulti` to get the same number of time steps as `TreeMesh`. 

This behavior does seem pretty counterintuitive, so it may be worth standardizing how we estimate CFL.

Reply:
Moved to #860

Reply:
Could you please increase the CFL number for this elixir here? The remaining parts will be discussed in #860.